.gitignore

```diff
@@ -3,3 +3,4 @@ venv
 build/
 dist/
 table_schema_to_markdown.egg-info/
+work/
```

Comment:
(You may add it to your own personal `.git/info/exclude` if it does not apply to other users)